src/test/ui/enum-discriminant/discriminant_value.rs

```diff
@@ -51,31 +51,31 @@ enum Mixed {
 }
 
 pub fn main() {
-    assert_eq!(discriminant_value(&CLike1::A), 0);
+    assert_eq!(discriminant_value(&CLike1::A), 0isize);
```

Comment:
Why these changes?

Reply:
To check that both the value and type of `discriminant_value` is correct. The test previously would not detect if we used a `i8` instead of the needed `i16` for a `#[repr(i16)]` enum.

Reply:
But you changed only some, not all of them.

Reply:
I only check the first occurrence for each type.